src/webview/wkwebview/mod.rs

```diff
@@ -645,15 +645,8 @@ impl Drop for InnerWebView {
         }
       }
 
-      // WKWebview has a single WKProcessPool to manage web contents.
-      // The WKProcessPool is not reset even if WKWebview is deallocated.
-      // So we need to override the process by navigating to `about:blank`.
-      self.navigate("about:blank");
-
-      let _: Id<_> = Id::from_ptr(self.webview);
-      #[cfg(target_os = "macos")]
-      let _: Id<_> = Id::from_ptr(self.ns_window);
-      let _: Id<_> = Id::from_ptr(self.manager);
+      let _: Id<_> = Id::from_retained_ptr(self.webview);
```

Comment:
Yep, tested it and it indeed cleans up the process we were talking about correctly! nice :partying_face: 

What it still doesn''t cleanup is the Networking process, but since it does the same on Linux it looks like a webkit problem :thinking:

Reply:
I think network connection is closed but network memory is still remained. I think this is because networking process is single process, networking memory is shared between multiple webview 👀

Reply:
> This process is responsible for handling network requests as well as storage management. All WebContent processes in a single session (default vs. private browsing) share a single networking session in the networking process.
> https://github.com/WebKit/WebKit/blob/main/Introduction.md

This is webkit description.
I think we can [remove cache](https://developer.apple.com/documentation/webkit/wkwebsitedatastore/1532938-removedataoftypes?language=objc) but all of webview cache will be removed. It brings to bad page loading performance.

Reply:
I will try to use [this gist](https://gist.github.com/dakeshi/6915b6681cbc29327dcc9b27cef263b8) to remove cache in another PR.